rss2email/config.py

```diff
@@ -59,11 +59,11 @@ def setup_html2text(self, section='DEFAULT'):
         """
         if section not in self:
             section = 'DEFAULT'
-        _html2text.UNICODE_SNOB = self.getboolean(
+        _html2text.config.UNICODE_SNOB = self.getboolean(
             section, 'unicode-snob')
-        _html2text.LINKS_EACH_PARAGRAPH = self.getboolean(
+        _html2text.config.LINKS_EACH_PARAGRAPH = self.getboolean(
             section, 'links-after-each-paragraph')
-        _html2text.BODY_WIDTH = self.getint(section, 'body-width')
```

Comment:
Does that mean these lines had no effect before?

Reply:
Correct. I assume they worked (were tested) when added and a change in html2text broke them. I was unable to find such a change.